mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -1,65 +1,101 @@
+            // but this could cause issues with target-specific code
+            // when we run the non-xdlops tests on a gfx900
+            matrix {
+                agent {
+                    docker {
+                        image 'rocm/mlir:miopen_test'
+                        args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+                        label "${XDLOPS == '1' ? 'gfx908 && multi_gpu' : 'gfx908 && multi_gpu'}"
+                        // The above could have gfx900 || gfx908 in the latter half
+                    }
+                }
+                axes {
+                    axis {
+                        name 'DTYPE'
+                        values 'fp16', 'fp32'
+                    }
+                    axis {
+                        name 'XDLOPS'
+                        values '1', '0'
+                    }
+                    axis {
+                        name 'LAYOUT'
+                        values 'NCHW', 'NHWC'
+                    }
+                    axis {
+                        name 'DIRECTION'
+                        values '1', '4'
+                    }
+                }
+                stages {
+                    stage('Install parallel (temp until dockerfile updated)') {
+                        steps {
+                            sh 'apt-get update && apt-get install -y --no-install-recommends parallel'
+                        }
+                    }
+                    stage('Environment') {
+                        steps {
+                            echo "Config is ($DTYPE, $LAYOUT, $XDLOPS, dir: $DIRECTION)"
+                            sh 'cat /etc/os-release'
+                            sh '/opt/rocm/bin/rocm-smi'
+                        }
+                    }
+                    stage('Build libMLIRMIOpen') {
+                        steps {
+                            checkout scm
+                            cmakeBuild generator: 'Unix Makefiles',\
+                               buildDir: 'build',\
+                               sourceDir: 'llvm',\
+                               installation: 'InDocker',\
+                               buildType: 'Release',\
+                               cmakeArgs: '''-DLLVM_ENABLE_PROJECTS="mlir;lld"
+                    -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
+                    -DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang
+                    -DBUILD_SHARED_LIBS=OFF
+                    -DLLVM_BUILD_LLVM_DYLIB=OFF
+                    -DLLVM_ENABLE_TERMINFO=OFF
+                    '''
+                            sh 'cd build; make -j $(nproc) libMLIRMIOpen'
+                        }
+                    }
+                    stage('Build MIOpenDriver') {
+                        steps {
+                            dir('MIOpen') {
+                                git branch: 'igemm-bin-xdlops', poll: false, url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
```

Comment:
You don't need to checkout to this branch anymore. You can leave it as the default develop branch.

Reply:
Done

Reply:
As shown in [build 16](http://10.236.106.97:8080/blue/organizations/jenkins/MLLIR-MIopen-Resnet50-refactor-dev/detail/MLLIR-MIopen-Resnet50-refactor-dev/16/pipeline/145) , using the `develop` branch causes the build to fail, so I've reverted the change. Please advise on how to fix this further.

Reply:
It must not be the branch that breaks it. The two branch should be equivalent because the xdlops PR is merged into develop and the clone does pull the latest commit of MIOpen.

Reply:
What error message points to is that the [post build action](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blob/miopen-dialect/mlir/tools/mlir-miopen-lib/CMakeLists.txt#L75-L80) from cmake you used to comment on is broken, in libMLIRMIOpen build.

Reply:
Fixed